drivers/wifi/siwx917/Kconfig.siwx917

```diff
@@ -44,4 +44,17 @@ config NET_MGMT_EVENT_STACK_SIZE
 config NET_MGMT_EVENT_QUEUE_SIZE
 	default 10
 
+# Override the WIFI_MGMT_SCAN_SSID_FILT_MAX parameter for the Wi-Fi subsystem.
+# This device supports filtering scan results for only one SSID.
+config WIFI_MGMT_SCAN_SSID_FILT_MAX
+	default 1
+
+# Override the WIFI_MGMT_SCAN_CHAN_MAX_MANUAL parameter for the Wi-Fi subsystem.
+# This device supports only the 2.4 GHz frequency band.
+config WIFI_MGMT_SCAN_CHAN_MAX_MANUAL
+	int "Maximum number of channels that can be manually specified"
+	range 1 14
+	default 3
+	help
+	  The default of 3 allows the 3 most common channels (2.4GHz: 1, 6, 11) to be specified.
```

Comment:
These option already exists. You don't have (you can't?) define `help`, `int` or `range`.

BTW, I don't see the purpose of changing the default of `WIFI_MGMT_SCAN_CHAN_MAX_MANUAL`